test/functional/test_framework/test_framework.py

```diff
@@ -247,6 +247,14 @@ def setup_nodes(self):
         self.add_nodes(self.num_nodes, extra_args)
         self.start_nodes()
 
+    def import_deterministic_coinbase_privkeys(self):
```

Comment:
Can you add a call to `import_deterministic_coinbase_privkeys()` to `setup_nodes()`? Would that reduce the number of tests that you need to change in this commit?

Reply:
Good idea to call this by default... This is now always called before `run_test`, unless `self.setup_clean_chain` is set.